stl/inc/xatomic.h

```diff
@@ -10,6 +10,9 @@
 #if _STL_COMPILER_PREPROCESSOR
 
 #include <intrin0.h>
+#if 1 // TRANSITION, GH-1197
+#include <intrin.h>
+#endif // TRANSITION, GH-1197

```

Comment:
I'd prefer to simply declare `_mm_pause` directly here. I'm preparing the fix for #1197 to apply internally, and once that's done - so we can be "certain" of the VS version it will ship in - I'll apply a change here. (And cleanup the TRANSITION below while I'm at it.)